### PR TITLE
Feature/non admin navigation permissions

### PR DESCRIPTION
**Pull request by mb-ca**
Added ability to retrieve user permissions in regards to a business.

### CODE REVIEW

**Comment by mb-ca** [comment]
@ryankazokas I can't figure out why the push build failed.  I attempt to get the userId from the session, but it's saying the the returned businessId is undefined.  This e2e test does work locally for me though.
Any thoughts?

```
it('Then return user business permissions', async () => {
          const businessId = '4baafa5b-4fbf-404e-9766-8a02ad45c3a4';
          const userId = await getUserId();
          const request = {
            businessId,
            userId
          };
          const permissions = await channelApeClient.businesses().getUserBusinessPermissions(request);
          expect(permissions.businessId).to.equal(businessId);
          expect(permissions.userId).to.equal(userId);
          expect(permissions.owner).to.be.a('boolean');
        });
```

It's failing here: `expect(permissions.businessId).to.equal(businessId);`
Stating `expected 'undefined' to equal '4baafa5b-4fbf-404e-9766-8a02ad45c3a4'`

File: src/businesses/service/BusinessesCrudService.ts [hunk 1]
```diff
@@ -77,4 +94,22 @@ export default class BusinessesCrudService {
     }
   }
 
+  private mapUserBusinessPermissionsPromise(
```
Comment (rjdavis3) [inline src/businesses/service/BusinessesCrudService.ts]: @mb-ca please fix this Sonar issue:
https://sonarcloud.io/project/issues?branch=feature%2FNon-Admin_Navigation_Permissions&id=channelape-typescript-sdk&resolved=false&severities=MAJOR&sinceLeakPeriod=true

File: e2e/ChannelApeClient.spec.ts [hunk 2]
```diff
@@ -739,35 +739,6 @@ describe('ChannelApe Client', () => {
         });
       });
     });
-
-    describe('And valid user business permissions get request', () => {
-      context('When getting user business permissions', () => {
```
Comment (ryankazokas) [inline e2e/ChannelApeClient.spec.ts]: @mb-ca can't you just add this back in with a different user Id?  You can even do it with the busienss owner: d87e6d12-d7c2-47f7-a7c6-a16b6fcb82f1 ?

**Comment by mb-ca** [comment]
@ryankazokas Corrections have been made.

**Comment by mb-ca** [comment]
@rjdavis3 Can you approve the corrections when you get a chance?

**Pull request merged**